### PR TITLE
Header footer mods

### PR DESCRIPTION
**Pull request by danielkryska**

### CODE REVIEW

**Comment by danielkryska** [comment]
image

**Comment by danielkryska** [comment]
image

**Comment by danielkryska** [comment]
Problem to be fixed:
image

The white row at the end of the footer with rights on larger screens `doesn't` cover the width of the screen. Only the width of the footer itself.

**Comment by Marcelinna** [comment]
image
we shouldn't add Copyright 2022 ?

**Comment by aleksandrahojszyk** [comment]
footer


header

https://www.figma.com/file/D2vgx66m1ADdBY0A46NdZ6/Search-Service?node-id=230%3A17089&t=gPU78ywj3c4YJ1Ec-1

@Marcelinna możesz sprawdzać implementacje :)

File: configurations/main-footer.development.config.json [hunk 1]
```diff
@@ -1,180 +1,190 @@
...
+        "label": "For providers",
+        "url": "https://eosc-portal.eu/for-providers",
+        "navBtns": [
+          {
+            "label": "Providers Documentation",
+            "url": "https://eosc-portal.eu/providers-documentation"
+          },
+          {
+            "label": "Promote your services",
+            "url": "#"
```
Comment (Michal-Kolomanski) [inline configurations/main-footer.development.config.json:75]: Is this URL correct? In other places, it is `"https://www.eosc-portal.eu/contribute-promotional-article-your-service"`

File: configurations/main-footer.development.config.json [hunk 2]
```diff
@@ -1,180 +1,190 @@
...
+            "label": "Submit your News/Event",
+            "url": "https://eosc-portal.eu/contribute-to-eosc"
+          },
+          {
+            "label": "Community",
+            "url": "https://eosc-portal.eu/documents/community"
+          },
+          {
+            "label": "Newsletters",
+            "url": "#"
```
Comment (Michal-Kolomanski) [inline configurations/main-footer.development.config.json:146]: Once again, is it correct? In other places it is: `"https://eosc-portal.eu/newsletters"`

**Review by Michal-Kolomanski — APPROVED** [review]
Let's wait for @Marcelinna or @agpul approve also

**Comment by Marcelinna** [comment]
image

**Comment by jarekzet** [comment]
added spacing mods, fixed social icons and links

**Review by michal-szostak — APPROVED** [review]

**Review by Marcelinna — APPROVED** [review]

**Pull request merged**